src/mann_whitney.jl

```diff
@@ -139,10 +139,11 @@ function pvalue(x::ExactMannWhitneyUTest; tail=:both)
         # cannot account for ties
         if tail == :both
             if x.U < x.nx * x.ny / 2
-                2 * pwilcox(x.U, x.nx, x.ny, true)
+                p = pwilcox(x.U, x.nx, x.ny, true)
             else
-                2 * pwilcox(x.U - 1, x.nx, x.ny, false)
+                p =  pwilcox(x.U - 1, x.nx, x.ny, false)
             end
+            min(2 * p, 1.0)
```

Comment:
I'm curious, why a `Float64` literal here?

Reply:
Probably no good reason. Just did it because I know `p` will be a `Float64`

Reply:
BTW below in the other branch `min(1, 2 * ...)` is used, i.e. no `Float64` literal.